splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/sequence/SpliceSequenceIT.java

```diff
@@ -79,13 +79,15 @@ public void testShortSequence() throws Exception {
         methodWatcher.executeUpdate("drop sequence SMALLSEQ restrict");
     }
 
-    @Ignore
+    @Test
     public void testIdentityValLocal() throws Exception{
         methodWatcher.executeUpdate(String.format("create table t1(c1 int generated always as identity, c2 int)"));
         methodWatcher.executeUpdate(String.format("insert into t1(c2) values (8)"));
         methodWatcher.executeUpdate(String.format("insert into t1(c2) values (IDENTITY_VAL_LOCAL())"));
 
-
+        methodWatcher.executeUpdate(String.format("create table t2(c1 int generated always as identity(start with 1, increment by 1), c2 int)"));
+        methodWatcher.executeUpdate(String.format("insert into t2(c2) values (8)"));
+        methodWatcher.executeUpdate(String.format("insert into t2(c2) values (IDENTITY_VAL_LOCAL())"));
```

Comment:
please add more data and check the results, see: https://db.apache.org/derby/docs/10.8/ref/rrefidentityvallocal.html